_overviews/scala3-book/fun-write-method-returns-function.md

```diff
@@ -116,27 +156,53 @@ A first thing you know is that you want to create a method that (a) takes a “d
 Furthermore, because that function prints a string that it’s given, you know it has the type `String => Unit`.
 With that information you write the method signature:
 
+{% tabs fun-write-method-returns-function-11 %}
+{% tab 'Scala 2 and 3' %}
 ```scala
 def createGreetingFunction(desiredLanguage: String): String => Unit = ???
 ```
+{% endtab %}
+{% endtabs %}
 
 Next, because you know that the possible functions you’ll return take a string and print it, you can write two anonymous functions for the English and French languages:
 
+{% tabs fun-write-method-returns-function-12 %}
+{% tab 'Scala 2 and 3' %}
 ```scala
 (name: String) => println(s"Hello, $name")
 (name: String) => println(s"Bonjour, $name")
 ```
+{% endtab %}
+{% endtabs %}
 
 Inside a method it might be a little more readable if you give those anonymous functions some names, so let’s assign them to two variables:
 
+{% tabs fun-write-method-returns-function-13 %}
+{% tab 'Scala 2 and 3' %}
 ```scala
 val englishGreeting = (name: String) => println(s"Hello, $name")
 val frenchGreeting = (name: String) => println(s"Bonjour, $name")
 ```
+{% endtab %}
+{% endtabs %}
 
 Now all you need to do is (a) return `englishGreeting` if the `desiredLanguage` is English, and (b) return `frenchGreeting` if the `desiredLanguage` is French.
 One way to do that is with a `match` expression:
 
+{% tabs fun-write-method-returns-function-14 class=tabs-scala-version %}
+{% tab 'Scala 2' for=fun-write-method-returns-function-14 %}
```

Comment:
We don’t need the `for` attribute anymore.

Reply:
Done with [9651d75](https://github.com/scala/docs.scala-lang/pull/2664/commits/9651d757bd5163e9b1bbbc311117aaa908d8209d).